keeper/msg_stableswap_server.go

```diff
@@ -351,6 +351,17 @@ func (s stableswapMsgServer) AddLiquidity(ctx context.Context, msg *stableswap.M
 		)
 	}
 
+	// Ensure that deposit amount of the base token is at least 1 unit (1e6).
+	if baseAmount.LT(math.LegacyNewDec(1e6)) {
```

Comment:
Should this be configurable? Like the base denom is?

Reply:
I thought about making it configurable, but since all pools are paired to `usdn` (baseAmount), and you always have to add a balanced amount of the pairing asset, just with this one control alone we are sure that the user will always deposit at least $1-$1.

Reply:
Yeah, I'm referring to adding a new `base_minimum_deposit` configuration parameter similar to `base_denom`.